v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,683 @@
+    }
+    requestBody.bruteForce = input.bruteForce;
+    requestBody.email = input.email;
+    requestBody.phoneNumber = input.phoneNumber;
+    requestBody.actionType = input.actionType;
+
+    let response = await axios.post(ANOMALY_DETECTION_API_URL, requestBody);
+
+    let responseData = response.data;
+    if (!responseData.bruteForce.result) {
+        return {
+            status: "GENERAL_ERROR",
+            message: "Too many requests. Please try again later."
+        }
+    }
+
+    if (responseData.isPasswordBreached) {
+        return {
+            status: "GENERAL_ERROR",
+            message: "This password has been detected in a breach. Please set a different password."
+        }
+    }
+
+    return undefined;
+}
+```
+
+### Overrides {#overrides}
+Here are a few examples of full implementations of the anomaly detection:
+
+#### Email and password
+<BackendSDKTabs>
+
+<TabItem value="nodejs">
+
+```tsx
+import SuperTokens from "supertokens-node";
+import EmailPassword from "supertokens-node/recipe/emailpassword";
+import Session from "supertokens-node/recipe/session";
+import axios from "axios";
+import { createHash } from 'crypto';
+
+const API_KEY = "<api-key>"; // Your secret API key that you received from the SuperTokens team
+const ANOMALY_DETECTION_API_URL = "https://security-us-east-1.aws.supertokens.io/v1/security";
+
+async function handleSecurityChecks(input: {
+    actionType?: string,
+    email?: string,
+    phoneNumber?: string,
+    password?: string,
+    requestId?: string,
+    bruteForce?: {
+        keyStr: string,
+        maxRequests: {
+            limit: number,
+            perTimeIntervalMS: number
+        }[]
+    }[]
+}): Promise<{
+    status: "GENERAL_ERROR",
+    message: string
+} | undefined> {
+    let requestBody: {
+        email?: string;
+        phoneNumber?: string;
+        actionType?: string;
+        requestId?: string;
+        passwordHash?: string;
+        bruteForce?: {
+            keyStr: string;
+            maxRequests: {
+                limit: number;
+                perTimeIntervalMS: number;
+            }[];
+        }[];
+    } = {}
+
+    if (input.requestId !== undefined) {
+        requestBody.requestId = input.requestId;
+    }
+
+    if (input.password !== undefined) {
+        let shasum = createHash('sha1');
+        shasum.update(input.password);
+        const passwordHash = shasum.digest('hex');
+        requestBody.passwordHash = passwordHash;
+    }
+    requestBody.bruteForce = input.bruteForce;
+    requestBody.email = input.email;
+    requestBody.phoneNumber = input.phoneNumber;
+    requestBody.actionType = input.actionType;
+
+    let response = await axios.post(ANOMALY_DETECTION_API_URL, requestBody);
+    let responseData = response.data;
+
+    if (!responseData.bruteForce.result) {
+        throw new Error("Too many requests. Please try again later.");
+    }
+
+    if (responseData.isBreachedPassword) {
+        // ... your code here ...
```

Comment:
instead of putting your code here, but actual code. We need to give people a sensible default impelmentation that they can just copy paste throughout

Reply:
return the error in the override

Reply:
done